src/components/AppBreadcrumbs/AppBreadcrumbs.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Breadcrumbs, Link } from '@material-ui/core';
+import { Link as RouterLink } from 'react-router-dom';
+
+function capitalize(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+const styles = (theme: any) => ({
+  breadcrumb: {
+    fontStyle: 'italic',
+  },
+  terminalBreadcrumb: {
+    fontStyle: 'italic',
+    color: theme.palette.primary.dark,
+    cursor: 'default',
+    '&:hover': {
+      textDecoration: 'none',
+    },
+  },
+});
+
+export type Context = { type: string; id: string; name: string };
+export type BreadcrumbElement = { text: string; to: string };
+
+type AppBreadcrumbsProps = {
+  links: Array<BreadcrumbElement>;
```

Comment:
I think it's a more common pattern to make this `children` and pass in the react `BreadcrumbsLink` or `TerminalBreadcrumb` components